### PR TITLE
[WIP]Move group interface

### PR DESCRIPTION
**Pull request by anasarrak**
Just ported Plan and execute functions, the only needed ones for the Demo

### CODE REVIEW

**Comment by LanderU** [comment]
OS X:

```bash
/tmp/ros2_ws/src/moveit2/moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp:161:5: error: no matching member function for call to 'waitForAction'
    waitForAction(move_action_client_, move_group::MOVE_ACTION, timeout_for_servers, allotted_time);
```

**Comment by anasarrak** [comment]
Linux passing, OSX same error as https://github.com/AcutronicRobotics/moveit2/pull/60#issuecomment-488581110

File: osx-compile-moveit2.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ cd /tmp
 source /tmp/ros2-osx/setup.bash
 mkdir -p /tmp/ros2_ws/src
 cp -r $dir /tmp/ros2_ws/src/moveit2
-cd /tmp/ros2_ws && wget https://raw.githubusercontent.com/AcutronicRobotics/moveit2/master/moveit2.repos
+cd /tmp/ros2_ws && wget https://raw.githubusercontent.com/AcutronicRobotics/moveit2/constraint_sampler_manager_loader_osx/moveit2.repos
```
Comment (LanderU) [inline osx-compile-moveit2.sh]: Make sure to change this before merge it.

Comment (ahcorde) [reply osx-compile-moveit2.sh]: Sure

**Comment by ahcorde** [comment]
CI is passing in MAC and Linux

**Pull request merged**